Added new event for Braze Currents (#900)
rudderlabs/rudderstack-docs: docs/sources/event-streams/cloud-apps/braze-currents.mdx

| Email Sent | An email was successfully sent to the user. |
| Email Delivered | An email was successfully delivered to the user. |
| Email Opened | The user opened an email. |
| Email Link Clicked | The user clicked a link within an email. Note that the email click tracking feature must be enabled. |
| Email Bounced | Triggered when Braze attempts to send an email but the receipient's email server does not accept it. |
| Email Soft Bounced | Triggered when Braze attempts to send an email but the recepient's email server bounces it temporarily because of reasons like full inbox, email server down, etc. |
| Email Marked As Spam | The user marked an email as spam. |
| Email Unsubscribed | The user clicked on the unsubscribe link within an email. |
| Subscription Group State Changed | The user's subscription group status changed to `Subscribed` or `Unsubscribed`. |
| Push Notification Sent | A push notification was successfully sent to the user. |
| Push Notification Tapped | The user tapped on a push notification. |
| iOS Foreground Push Opened | The user received an iOS push notification while the app was open. |
| In-App Message Viewed |The user viewed an in-app message. |

> | In-App Message Clicked |This event occurs when a user clicks on an in-app message. |

Reviewer: ```suggestion
| In-App Message Clicked |The user clicked an in-app message. |
```
